pyproject.toml

```diff
@@ -121,6 +121,7 @@ source-includes = [
 name = "fastapi-slim"
 
 [tool.mypy]
+plugins = ["pydantic.mypy"]
```

Comment:
Is this necessary?

Reply:
This is what enabled the suppressions in fastapi/security/api_key.py to be removed.